detections/network/windows_remote_desktop_network_bruteforce_attempt.yml

```
@@ -0,0 +1,61 @@
+name: Windows Remote Desktop Network Bruteforce Attempt
+id: 908bf0d5-0983-4afd-b6a4-e9eb5d361a7d
+version: 6  
+date: '2025-01-10'
+author: Jose Hernandez, Bhavin Patel, Splunk
+status: production
+type: TTP
```

In my opinion and in order to avoid folding the users at first. Perhaps we could start with Anomaly and see how it performs in real life via telemetry stats? 

```suggestion
type: Anomaly
```